### PR TITLE
Adding db into Pay Desk

### PR DESCRIPTION
**Pull request by sprassanna**
Added hibernate and PostgreSQL configuration in pay desk , to test the DB part. 

UI for Data Base: PDADMIN III 

image

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -9,6 +9,9 @@
 	<properties>
 		<spring.version>4.0.1.RELEASE</spring.version>
 		<java.version>1.8</java.version>
+		<hibernate.version>5.2.2.Final</hibernate.version>
```
Comment (ghost) [inline pom.xml:12]: Why dont we put this also as property and set ?

File: src/main/java/com/paydesk/data/AppUser.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+import java.io.Serializable;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class AppUser implements Serializable {
```
Comment (ghost) [inline src/main/java/com/paydesk/data/AppUser.java]: A little better name ? AppUser looks more of app related data may be UserRole ?

**Comment by ghost** [comment]
Looks good

**Comment by MithileshAnna** [comment]
Looks Good. :+1:

**Comment by Saravanagithub** [comment]
Looks Good :+1:

File: pom.xml (hunk 1 above)
Comment (sprassanna) [reply pom.xml:12]: Still not getting it :)

**Comment by sprassanna** [comment]
Updated review comments

**Comment by ghost** [comment]
Looks Good :+1:

**Pull request merged**